app/controllers/api/v3/statements_controller.rb

```diff
@@ -1,6 +1,10 @@
 module API
   module V3
     class StatementsController < BaseController
+      include ::API::Concerns::FilterByUpdatedSince
```

Comment:
maybe this can be a service concern rather that in the controller?

Reply:
The concern should be located `app/controllers/concerns/api/filter_by_updated_since.rb`
That way the include would be:
```ruby
include FilterByUpdatedSince
```